fluidBuild.config.cjs

```diff
@@ -71,8 +71,8 @@ module.exports = {
 			dependsOn: ["build:esnext"],
 			script: true,
 		},
-		"build:docs": ["tsc"],
-		"ci:build:docs": ["tsc"],
+		"build:docs": ["tsc", "build:esnext"],
+		"ci:build:docs": ["tsc", "build:esnext"],
```

Comment:
So, should projects have "build:docs" if they have "api-extractor:.."?
Seems like the main job of "build:docs" is to generate the report and primary api-extractor also does that, right? So, they are in competition.?
If so, we should remove "build:docs" script entries from those packages? (Is there way to configure so that "build:docs" dependencies become "api" dependencies?

Reply:
I see some places use
```json
"build:docs": "fluid-build . --task api",
```
That seems like it will do.